config/templates/managed-velero-operator-csv-template.yaml

```diff
@@ -0,0 +1,50 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: managed-velero-operator-0.0.1
+  namespace: placeholder
+  annotations:
+    categories: A list of comma separated categories that your operator falls under.
+    certified: "false"
+    description: Operator to manage installation of Velero in OpenShift.
+    containerImage: quay.io/app-sre/managed-velero-operator:latest
+    createdAt: "2019-11-11T11:00:00Z"
+    support: Stefanie Forrester
+spec:
+  displayName: Managed Velero Operator
+  description: Operator to manage installation of Velero in OpenShift.
+  keywords:
+  - kubernetes
+  - backup
+  - openshift
+  - velero
+  - storage
+  version: 0.0.1
+  provider:
+    name: Red Hat, Inc
+  maturity: alpha
+  installModes:
+  - type: OwnNamespace
+    supported: true
+  - type: SingleNamespace
+    supported: true
```

Comment:
I think this should be false.. the OperatorGroup should exist in `openshift-velero`

Reply:
I could be wrong, but I imagine this setting is correct because the other operators seem to have this same setting:

```
  installModes:
  - type: OwnNamespace
    supported: true
  - type: SingleNamespace
    supported: true
  - type: MultiNamespace
    supported: false
  - type: AllNamespaces
    supported: false
```

I think it just means that the operator is scoped to a namespace instead of being scoped cluster-wide. Let me know if I'm missing something. I didn't really understand how the `OperatorGroup` affects this. But I will be working with James on Monday to deploy this, so I'll run it by him too.